addons/pause/addon.json

```diff
@@ -30,10 +28,27 @@
       "matches": ["projects", "projectEmbeds"]
     }
   ],
+  "settings": [
+    {
+      "type": "boolean",
+      "id": "pause-button",
+      "name": "Show pause button",
```

Comment:
This setting doesn't work correctly (the button always shows after the project loads) and I doubt users will disable it.